__tests__/index.test.js

```diff
@@ -21,8 +22,21 @@ describe("/api/topics", () => {
         });
       });
   });
+});
+
+describe("/api", () => {
+  test("GET 200: Responds with the contents of the file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints);
+      });
+  });
+});
 
-  test.only("GET 404: Responds with a 404 error if not found", () => {
+describe("returns an error when the file is not found", () => {
```

Comment:
I'd look at this description, is there a file causing this error? I'd argue this is an error for an endpoint that isn't found.